Pipfile

```
@@ -6,6 +6,8 @@ verify_ssl = true
 [packages]
 "discord.py" = "~=1.5.1"
 loguru = "*"
+fuzzywuzzy = "*"
+python-levenshtein = "*"
```

Instead of using `*` version, stick to a definite one, like discord.py.
```suggestion
fuzzywuzzy = "~=0.18.0"
python-levenshtein = "~=0.12.0"
```
Also i can't see where `python-levenshtein` is being used.